core/src/main/scala/org/apache/spark/util/Utils.scala

```diff
@@ -3084,6 +3085,35 @@ private[spark] object Utils extends Logging {
     conf.get(config.STORAGE_BLOCKMANAGER_HEARTBEAT_TIMEOUT)
       .getOrElse(Utils.timeStringAsMs(s"${conf.get(Network.NETWORK_TIMEOUT)}s"))
   }
+
+  /** Decompress a zip file into a local dir. File names are read from the zip file. */
```

Comment:
Looks like this is not the Java doc style we follow.

Reply:
We have been using one-liner java doc; e.g. https://github.com/apache/spark/blob/master/sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/state/StateStore.scala#L50

Reply:
Oh, I thought it is two lines.

Reply:
But we'd like to see this multiple lines as below review comments :)

Reply:
yea :)

Reply:
Yea, we'll have multiple lines here in the next commit :)

Reply:
@xuanyuanking Seems you overwrite previous change? This looks the previous version.

Reply:
@xuanyuanking friendly reminder

Reply:
……Sorry I missed this... Updating